### PR TITLE
Disable experimental REST API by default

### PR DESCRIPTION
**Pull request by mik-laj**
We have a new REST API, so we can disable the experimental REST API because it is unsecure.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/config_templates/config.yml [hunk 1]
```diff
@@ -704,6 +704,22 @@
...
+        The authenticated user has full access.
+
+        .. warning::
+
+          This `Experimental REST API <https://airflow.readthedocs.io/en/latest/rest-api-ref.html>`__ is
+          deprecated since version 2.0. Please consider using
+          `the Stable REST API <https://airflow.readthedocs.io/en/latest/stable-rest-api-ref.html>`__.
+          For more information on migration, see
+          `UPDATING.md <https://github.com/apache/airflow/blob/master/UPDATING.md>`_
+      version_added: ~
```
Comment (kaxil) [inline airflow/config_templates/config.yml]:
```suggestion
      version_added: 2.0.0
```

**Comment by kaxil** [comment]
Kerbereos tests are failing

**Comment by mik-laj** [comment]
I pushed fix-up.

**Comment by kaxil** [comment]
All Kubernetes tests are failing

**Comment by kaxil** [comment]
```
kubernetes_tests/test_kubernetes_executor.py:209: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
kubernetes_tests/test_kubernetes_executor.py:195: in start_job_in_kubernetes
    result_json = self.start_dag(dag_id=dag_id, host=host)
kubernetes_tests/test_kubernetes_executor.py:168: in start_dag
    self.assertEqual(result.status_code, 200, f"Could not enable DAG: {result_json}")
E   AssertionError: 404 != 200 : Could not enable DAG: <Response [404]>
----------------------------- Captured stdout call -----------------------------
Calling [start_dag]#1 http://localhost:8080/api/experimental/dags/example_kubernetes_executor_config/paused/false
Received [start_dag]#1 <Response [404]>
____ TestKubernetesExecutor.test_integration_run_dag_with_scheduler_failure ____
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**